src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(x) {
+      calculator.result += x;
+    },
+    subtract(x) {
+      calculator.result -= x;
+    },
+    multiply(x) {
+      calculator.result *= x;
+    },
+    divide(x) {
+      calculator.result /= x;
+    },
```

Comment:
It is necessary to provide for a check on 0, since it is not possible to divide by 0